### PR TITLE
Group 4- Tests passing so far

### PR DESCRIPTION
**Pull request by gooca**
Nektarios
Charlie
Nick

### CODE REVIEW

File: language_detector/main.py [hunk 1]
```diff
@@ -2,10 +2,24 @@
...
+            print("this is text", text, "this is word", word)
+            return True
+    return False
+    
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
```
Comment (pkkondamuri) [inline language_detector/main.py]: Not sure why you retained 'pass'

File: language_detector/main.py [hunk 2]
```diff
@@ -2,10 +2,24 @@
 
 """This is the entry point of the program."""
 
-from languages import LANGUAGES
+from .languages import LANGUAGES
+
+def detectInOneLanguage(text,lang):
```
Comment (adoniasdantas) [inline language_detector/main.py:7]: Did you consider the case of cognate words in the text? For example, German and English have some cognates words (all, has, die, so). Case you put a german text and it get the 'all' word, you could get 'English' as return of the function.

File: language_detector/main.py [hunk 3]
```diff
@@ -2,10 +2,24 @@
...
+            return True
+    return False
+    
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    for lang in LANGUAGES:
```
Comment (adoniasdantas) [inline language_detector/main.py:20]: Great code! Clean and easy to read. Congrats!

File: language_detector/main.py (hunk 2 above)
Comment (pkkondamuri) [reply language_detector/main.py:7]: I agree with Adonias. This code as it is would return incorrect language in some cases. A better logic would be to count the number of common words from each language that appear in the given text and pick the language that has the highest count.

Comment (eyezick) [reply language_detector/main.py:7]: Agree with above, including logic involving the frequency of the common words (such as highest count) would better ensure an accurate result

**Pull request closed without merging**